Mathlib/Topology/Algebra/Module/LinearMap.lean

```diff
@@ -126,7 +126,7 @@ def Simps.apply (h : M₁ →SL[σ₁₂] M₂) : M₁ → M₂ :=
 def Simps.coe (h : M₁ →SL[σ₁₂] M₂) : M₁ →ₛₗ[σ₁₂] M₂ :=
   h
 
-initialize_simps_projections ContinuousLinearMap (toFun → apply, toLinearMap → coe)
+initialize_simps_projections ContinuousLinearMap (toFun → apply, toLinearMap → coe, as_prefix coe)
```

Comment:
I'm baffled by this change. What does it do? Excuse my ignorance :-/ Essentially nothing else in this file is changed. There is also a random change of a section name. What do the changes in this file have to do with the main point of this PR?

Reply:
This makes `@[simps! coe apply] def coprod` generate `coe_coprod` and `coprod_apply` rather than `coprod_coe` and `coprod_apply`. This change is just here to avoid me having to write `coe_coprod` by hand

Reply:
Even if `coprod` is defined in a completely different file?

Reply:
Yeah, I tested and you're right! I didn't know `initialize_simps_projections` had such a scope!